src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,106 @@
+package tasks;
+
+/**
+ * Represents a general task with a name and completion status.
+ */
+public class Task {
+    protected String taskName;
+    protected boolean status;
```

Comment:
I think it will better to name boolean values in the form of isDone to make it clearer.